rtengine/imagefloat.h

```diff
@@ -109,35 +109,35 @@ class Imagefloat final : public IImagefloat, public ImageIO
         } tmp;
 
         tmp.f = f;
-        int32_t sign     =  (tmp.i >> 16) & 0x00008000;
+        const int32_t lsign = (tmp.i >> 16) & 0x00008000;
```

Comment:
`std::int32_t`

Reply:
We should refactor this stuff, as it's duplicated code. Have a look:
https://github.com/Beep6581/RawTherapee/blob/dev/rtengine/dcraw.cc#L10709